lib/assert.js

```diff
@@ -49,7 +50,8 @@ const { isPromise, isRegExp } = require('internal/util/types');
 const { EOL } = require('internal/constants');
 const { NativeModule } = require('internal/bootstrap/loaders');
 const { isError } = require('internal/util');
-
+const util = require('util');
+const console = require('console');
```

Comment:
`util.inspect` is already imported as its own function from line 47/48 here, and I would move the `console` import into the `runCallChecks` function maybe so that it only is loaded when needed?